src/validation/validation-environment-maker.js

```diff
@@ -1,6 +1,6 @@
 /**
```

Comment:
Renamed to provide consistency of terminology between this module and the test-environment-maker module.